R/Filter.R

```diff
@@ -344,6 +360,65 @@ makeFilter(
   }
 )
 
+# information.gain ----------------
+
+#' Filter \dQuote{information.gain} uses the entropy-based information gain
+#' between each feature and target individually as an importance measure.
+#'
+#' @rdname makeFilter
+#' @name makeFilter
+makeFilter(
+  name = "information.gain",
+  desc = "Entropy-based information gain between feature and target",
+  pkg  = "FSelector",
+  supported.tasks = c("classif", "regr"),
+  supported.features = c("numerics", "factors"),
+  fun = function(task, nselect, ...) {
+    y = FSelector::information.gain(getTaskFormula(task), data = getTaskData(task))
```

Comment:
Wait, either I'm missing something here or you're using the wrong functions -- shouldn't this be `FSelectorRcpp::` instead of `FSelector::`?

Reply:
The purpose is to re-add the removed FSelector filters because of [bernds comment](https://github.com/mlr-org/mlr/pull/2140#issuecomment-435698491). 

Because FSelector != FSelectorRcpp.

The FSelectorRcpp funs are already merged.

Reply:
Ah yes, sorry!